tools/debian/control

```diff
@@ -28,24 +30,26 @@ Build-Depends: debhelper (>= 9),
                glib-networking,
                nodejs-legacy,
                npm,
-               openssh-client
-Standards-Version: 3.9.6
+               openssh-client <!nocheck>,
+Standards-Version: 3.9.8
 Homepage: http://cockpit-project.org/
 #Vcs-Git: git://anonscm.debian.org/collab-maint/cockpit.git
 #Vcs-Browser: https://anonscm.debian.org/cgit/collab-main/cockpit.git
 
 Package: cockpit
-Architecture: any
+Architecture: all
 Depends: ${misc:Depends},
-         cockpit-bridge (= ${binary:Version}),
-         cockpit-ws (= ${binary:Version}),
+         cockpit-bridge (>= ${source:Version}),
+         cockpit-bridge (<< ${source:Version}.1~),
+         cockpit-ws (>= ${source:Version}),
+         cockpit-ws (<< ${source:Version}.1~),
```

Comment:
I believe the cockpit meta package should enforce that identical versions of the subpackages are installed. If not, then why not? If someone wants to mix and match parts of cockpit, then the cockpit meta package is something that should be removed.